### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +95,7 @@ Fizzy passes all of the official WebAssembly 1.0 tests. We are maintaining the W
 with corrections and additions backported from the WebAssembly specification master branch. For this
 Fizzy release [the snapshot from 2021-02-12](https://github.com/wasmx/wasm-spec/tree/w3c-1.0-tests-backported-20210212/test/core) is used:
 - 19044 of 19044 binary parser and execution tests,
-- 1009 of 1009 validation tests,
+- 1049 of 1049 validation tests,
```
Comment (gumb0) [inline CHANGELOG.md:98]: This was apparently miscalculation last time.

**Review by chfast — APPROVED** [review]
I added benchmark results so please check these.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,178 @@ and this project adheres to [Semantic Versioning].
...
+- 499 skipped due to testing text format parser.
+
+Comparing with the previous release, the performance of instantiation remains unchanged (within 1% performance regression). 
+However, we were able to increase execution performance by around **7%**.
+
+<details><summary>Detailed benchmark results</summary>
+
+```
+Comparing Fizzy 0.6.0 to 0.7.0 (Intel Haswell CPU 4.4 GHz, Clang 14)
```
Comment (axic) [inline CHANGELOG.md:23]: 0.6.0 to 0.7.0?

@chfast This is the changelog for 0.8.0.

Comment (gumb0) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,178 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.8.0] - unreleased
```
Comment (axic) [inline CHANGELOG.md:9]:
```suggestion

With this release we are introducing support for runtime metering (i.e. deterministic execution). This is only available via the C API.

Additionally we have made a lot of improvements to both the C and Rust APIs.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,181 @@ and this project adheres to [Semantic Versioning].
...
 
+With this release we are introducing support for runtime metering (i.e. deterministic execution). This is only available via the C API.
+
+Additionally we have made a lot of improvements to both the C and Rust APIs.
```
Comment (axic) [inline CHANGELOG.md:12]: Missed a new line:
```suggestion
Additionally we have made a lot of improvements to both the C and Rust APIs.

```

**Review by axic — APPROVED** [review]

**Pull request merged**